sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -189,7 +189,7 @@ class APISpec extends Specification {
     }
 
     static boolean liveMode() {
-        return setupTestMode() == TestMode.RECORD
+        return setupTestMode() == TestMode.RECORD && false
```

Comment:
What does this mean?

Reply:
I was forcing the live test only tests not to record playback records while I was updating the other tests, removed.